### ISSUE TITLE
Feat -> Add Internationalisation -> Default English

### ISSUE THREAD

**Issue by ekas** [jslovers/jslovers-official-website#5]

**Comment by purohitdheeraj** [comment]
Hello @ekas i can do this, please assign this to me

what are requirements ?
- which i18n library ( react-intl , next-i18next, or i18next) go with
- any component to add ? or just config part
- any specific direction you want to give 

thank you!

**Comment by ekas** [comment]
> Hello @ekas i can do this, please assign this to me
> 
> what are requirements ?
> 
> * which i18n library ( react-intl , next-i18next, or i18next) go with
> * any component to add ? or just config part
> * any specific direction you want to give
> 
> thank you!

Go for next-i18next, Just initial config and add one example with docs in contributing. Use the Namespace technique, every page should have a separate file. Let's talk on the discord #website channel if you have more detailed doubts.

I would appreciate it if you could achieve this quickly. We could review your PR on priority as this is an essential section. Thanks for taking it.

**Comment by Neha** [comment]
Hello @purohitdheeraj  would you be going ahead with this issue?

**Comment by djhemath** [comment]
If this is still open, I can pick it up

**Comment by Neha** [comment]
Thank you @djhemath . For now, we need to see the scale of this and do we need this? Please see if there is another issue you can pick

**Comment by djhemath** [comment]
Yeah, sure I picked this because I've done i18n in the past. I'll look for some other issues that I can contribute.
